### PR TITLE
Silabs EFR32 codelab first draft to publish for openthread.io

### PR DESCRIPTION
**Pull request by silabs-mithil**
Add EFR32 codelab to openthread.io
Please validate the preview on openthread.io and let me know of any issues or errors in rendering.

### CODE REVIEW

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 1]
```diff
@@ -0,0 +1,438 @@
...
+
+> For convenience, I also right clicked and renamed the debug adapters as `ot-cli-ftd 1` and `ot-cli-ftd 2`. 
+
+![FTD Studio Console View](./img/Silabs/console_ftds.png)
+
+You will notice that we do not yet have a console for `ot-rcp`. In the next step we will configure the Raspberry Pi as an OT border router and setup the console for `ot-rcp`.
+
+
+
+## Setup the Raspberry Pi as a Border Router
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:255]: "Set up" when using it as a verb

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 2]
```diff
@@ -0,0 +1,438 @@
...
+
+https://hub.docker.com/r/siliconlabsinc/openthread-backbone-border-router/tags
+
+### Raspberry Pi setup
+
+1. On your SD card, make sure you flash the  [Raspbian Stretch Lite OS image](http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip) or [Raspbian Stretch with Desktop](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip).
+2. You can either SSH onto the Raspberry Pi or choose to work with the Raspbian Desktop directly. Open a terminal.
+3. Make sure to update your local repositories and package manager (**apt-get update** and **apt-get upgrade** prior to installing Docker).
+
+NOTE: Make sure to reboot the Raspberry Pi after any updates.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:273]: > Note: Make sure....

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 3]
```diff
@@ -0,0 +1,438 @@
...
+* How to verify Thread communication between the nodes using the ping command.
+
+## Further reading
+
+Check out  [openthread.io](https://openthread.io) and [GitHub](https://github.com/openthread) for a variety of OpenThread resources, including:
+
+*  [Supported Platforms](https://openthread.io/platforms/)
+    — discover all the platforms that support OpenThread
+    
+*  [Build OpenThread](https://openthread.io/guides/build/)
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:422]: For files that are in this repo, we're using relative links now

../../guides/build/index.md

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 4]
```diff
@@ -0,0 +1,438 @@
...
+    — further details on building and configuring OpenThread
+    
+*  [Thread Primer](https://openthread.io/guides/thread-primer/)
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:425]: ../../guides/thread-primer/index.md

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 5]
```diff
@@ -0,0 +1,438 @@
...
+* How to create an OpenThread project using Silicon Labs Simplicity Studio IDE.
+* How to build and flash OpenThread CLI binaries to Silicon Labs radio boards.
+* How to setup a Raspberry Pi 3B+ or greater as an OpenThread Border Router (OTBR) using Docker.
+* How to create a Thread network on the OTBR.
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+## Further reading
+
+Check out  [openthread.io](https://openthread.io) and [GitHub](https://github.com/openthread) for a variety of OpenThread resources, including:
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:417]: Can you add a / at the end of the link, needed for our import process

https://openthread.io/

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 6]
```diff
@@ -0,0 +1,438 @@
...
+[Codelab Feedback](https://github.com/openthread/openthread/issues)
+
+
+## Introduction
+
+
+
+<img src="img/Silabs/26b7f4f6b3ea0700.png" alt="26b7f4f6b3ea0700.png" width="624.00" />
+
+Google's [OpenThread](https://openthread.io) (OT) is an open-source implementation of Thread. Google has released OpenThread to make the networking technology used in Google Nest products more broadly available to developers, in order to accelerate the development of products for the connected home and commercial buildings. With a narrow platform abstraction layer and a small memory footprint, OpenThread is highly portable. It supports both system-on-chip (SoC) and network co-processor (NCP) designs.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:51]: Add a / at the end of the link for our import process

https://openthread.io/

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 7]
```diff
@@ -0,0 +1,438 @@
...
+ *  CONSEQUENTIAL DAMAGES (INCLUDING, BUT NOT LIMITED TO, PROCUREMENT OF
+ *  SUBSTITUTE GOODS OR SERVICES; LOSS OF USE, DATA, OR PROFITS; OR BUSINESS
+ *  INTERRUPTION) HOWEVER CAUSED AND ON ANY THEORY OF LIABILITY, WHETHER IN
+ *  CONTRACT, STRICT LIABILITY, OR TORT (INCLUDING NEGLIGENCE OR OTHERWISE)
+ *  ARISING IN ANY WAY OUT OF THE USE OF THIS SOFTWARE, EVEN IF ADVISED OF THE
+ *  POSSIBILITY OF SUCH DAMAGE.
+ */
+
+---
+id: openthread-hardware
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:30]: id: silabs-openthread-hardware

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 8]
```diff
@@ -0,0 +1,438 @@
...
+authors: Mithil Raut
+categories:
+tags:
+feedback link: https://github.com/openthread/openthread/issues
+
+---
+
+# Build a Thread network with Silicon Labs EFR32 boards and OpenThread using Simplicity Studio v5
+
+[Codelab Feedback](https://github.com/openthread/openthread/issues)
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:42]: https://github.com/openthread/ot-docs/issues

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 9]
```diff
@@ -0,0 +1,438 @@
...
+
+1. EFR32MG12 [BRD4166A Thunderboard Sense 2](https://www.silabs.com/development-tools/thunderboard/thunderboard-sense-two-kit) as shown on the left.
+2. Two EFR32MG13 [BRD4168A](https://www.silabs.com/documents/public/user-guides/ug280-brd4168a-user-guide.pdf) as shown on the right. 
+3. Simplicity Studio v5 installed on a macOS Catalina 10.15.7 with
+   1. Gecko SDK 3.1.2 
+   2. GNU ARM v7.2.1
+
+ ![](img/Silabs/BRD4166A.jpeg)![BRD4168A](./img/Silabs/BRD4168A.jpeg)
+
+> You can update the GSDK or toolchain from Studio -> Help -> Update Software -> SDKs/Toolchains
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:125]: > Note: You can update...

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 10]
```diff
@@ -0,0 +1,438 @@
...
+
+Connect each Wireless Starter Kit main board to the host computer by USB as shown in the following figure. These connections will permit programming and network analysis of the RCP and end devices. Optionally, end devices may be connected to the host computer via a common Ethernet switch. The starter kits also support programming and network analysis over IPv4.
+
+![Connections](./img/Silabs/connections.jpg)
+
+### Firmware setup
+
+> There are two ways to get started. Both options allow you to flash the firmwares that are required for this codelab.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:133]: > Note: There are...

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 11]
```diff
@@ -0,0 +1,438 @@
...
+
+If you use the Demos option to flash the radio boards then you can skip the next section about creating a project.
+
+#### Creating a Project Based on an Example
+
+Duration: 10 minutes
+
+We will create two projects. The ot-rcp project for BRD4166A and the ot-cli-ftd project for the two BRD4168A. Please follow these steps and select the appropriate sample application for your board.
+
+1. Open Studio's File menu and select New > Silicon Labs Project Wizard. The Target, SDK, and Toolchain Selection dialog opens. Do not change the default Simplicity IDE / GNU toolchain supported by OpenThread. Click Next.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:183]: For all buttons and menu options, use bold:

**New** > **Silicon Labs Project Wizard**

Please update elsewhere names of controls like this are used.

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 12]
```diff
@@ -0,0 +1,438 @@
...
+
+   Target boards: Shows the selected radio board (BRD4168A) along with the main board (BRD4001A)
+
+   Target Device: This field shows the microcontroller chip (MCU) onboard. The BRD4168A has an EFR32MG13 MCU onboard.
+
+   SDK: Here you can select the SDK version of OT that you are working with. For this codelab we select OpenThread 1.1.2.0
+
+   IDE/ Toolchain: The toolchain that will be used for compiling the OT project. We use GNU ARM 7.2.1
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:191]: These should be formatted as an indented list under the 1. bullet:

* Target boards: Shows the selected radio board (BRD4168A) along with the main board (BRD4001A)
* Target Device: This field shows the microcontroller chip (MCU) onboard. The BRD4168A has an EFR32MG13 MCU onboard.
* SDK: Here you can select the SDK version of OT that you are working with. For this codelab we select OpenThread 1.1.2.0
* IDE/ Toolchain: The toolchain that will be used for compiling the OT project. We use GNU ARM 7.

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 13]
```diff
@@ -0,0 +1,438 @@
...
+
+   > The IAR Embedded Workbench for ARM (IAR EWARM) Compiler is currently not supported with OpenThread.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:193]: Do not indent and format as a note

> Note: The IAR...

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 14]
```diff
@@ -0,0 +1,438 @@
...
+ping fd5c:c6b:3a17:40b9:0:ff:fe00:1800
+Done
+> 
+> 16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=30ms
+16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=52ms
+```
+
+ The response indicates the payload received and that the communication is successful. Repeat the process to ping FTDs from the OTBR.
+
+**You've created a Thread network!**
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:404]: Let's add a level 2 Congratulations! header here, and have that be the last step.

Further Reading can be a level 3 section under it.

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 15]
```diff
@@ -0,0 +1,438 @@
...
+
+  * GNU ARM v7.2.1 toolchain 
+
+  * Gecko SDK Suite 3.1.0 or later and Silicon Labs OpenThread SDK.
+
+    > The IAR Embedded Workbench for ARM (IAR EWARM) Compiler is currently not supported with OpenThread.
+
+
+
+## Getting started
```
[the remark this answers is not in the document]
Comment (silabs-mithil) [reply site/en/codelabs/silabs-openthread-hardware/index.lab.md]: > This step is quite lengthy, I suggest we break it up into the following so it's more digestible:
> 
> Hardware setup
> Firmware setup - demos
> Firmware setup - projects
> 
> Ideally in a codelab the user shouldn't jump around or skip steps, though, so it would be better to just have one setup method. Would you be willing to just have one method of firmware setup and remove the other, leaving it as an exercise for the reader?

@Vyrastas  I have reorganised the secion into Hardware and Firmware setup.  Firmware setup - demos is put as an optional exercise in the end. Is it possible to link this optional section in the earlier step under Firmware set up point 2 Demos?

**Comment by silabs-mithil** [comment]
@Vyrastas Thank you for all the suggestions. I have revised the draft to incorporate them.

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 16]
```diff
@@ -0,0 +1,438 @@
+/*
+ *  Copyright (c) 2021, The OpenThread Authors.
```
[the remark this answers is not in the document]
Comment (silabs-mithil) [reply site/en/codelabs/silabs-openthread-hardware/index.lab.md]: @Vyrastas , I have moved the license. It says the copyright belongs to the OpenThread Authors. I am trying to understand if it applies for this codelab or should I rephrase it to something like the copyright belongs to Silicon Labs?

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 17]
```diff
@@ -0,0 +1,461 @@
...
+* Out of band commissioning of devices onto a Thread network.
+* How to verify Thread communication between the nodes using the ping command.
+
+## Prerequisites
+
+Duration: 3:00
+
+Hardware:
+
+1. 3 EFR32MGxx radio boards - you can use any combination of these devices. I used a BRD4166A as the RCP and two BRD4168As as Full Thread Devices.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:51]: Don't refer to yourself in the text, use something like "this codelab uses a BRD4166A..."

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 18]
```diff
@@ -0,0 +1,461 @@
...
+
+Software:
+
+* [Simplicity Studio v5](https://www.silabs.com/developers/simplicity-studio) installed and updated on the Windows/Linux/Mac Host system with
+
+  * GNU ARM v7.2.1 toolchain 
+
+  * Gecko SDK Suite 3.1.0 or later and Silicon Labs OpenThread SDK.
+
+    > Note: The IAR Embedded Workbench for ARM (IAR EWARM) Compiler is currently not supported with OpenThread.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:79]: Add this before the note, don't indent this note.

<noparse>> aside positive</noparse>
<noparse>></noparse>

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 19]
```diff
@@ -0,0 +1,461 @@
...
+
+1. EFR32MG12 [BRD4166A Thunderboard Sense 2](https://www.silabs.com/development-tools/thunderboard/thunderboard-sense-two-kit) as shown on the left.
+2. Two EFR32MG13 [BRD4168A](https://www.silabs.com/documents/public/user-guides/ug280-brd4168a-user-guide.pdf) as shown on the right. 
+3. Simplicity Studio v5 installed on a macOS Catalina 10.15.7 with
+   1. Gecko SDK 3.1.2 
+   2. GNU ARM v7.2.1
+
+ ![](img/BRD4166A.jpeg)![BRD4168A](./img/BRD4168A.jpeg)
+
+> Note: You can update the GSDK or toolchain from **Studio -> Help -> Update Software -> SDKs/Toolchains**
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:97]: Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 20]
```diff
@@ -0,0 +1,461 @@
...
+
+## Firmware set up summary
+
+Duration: 1:00
+
+At this point you should have created, compiled and flashed the appropriate firmware on the radio boards. Once `ot-rcp` is flashed to BRD4166A, disconnect it from the host system and connect this board to the Raspberry Pi.
+
+After completing this section, your Thread network hardware set up will look like this.
+
+> Not shown in the image is the WSTKs connected to a Windows/Linux/Mac Host system via USB and the Raspberry Pi 3B+ is connected to the internet using an Ethernet cable.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:177]: Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 21]
```diff
@@ -0,0 +1,461 @@
...
+
+
+
+## Set up the Serial console for `ot-cli-ftd` devices
+
+Duration: 1:00
+
+To launch the Console interface, in the Simplicity IDE perspective **right-click on your J-Link device** in the Devices View / Debug Adapters Window. Choose **Launch Console**. To get a prompt on the Console, choose **Serial 1** tab and **press Enter**. Check the state of the FTD node.
+
+> Note: For convenience, I also right clicked and renamed the debug adapters as `ot-cli-ftd 1` and `ot-cli-ftd 2`.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:189]: Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 22]
```diff
@@ -0,0 +1,461 @@
...
+
+https://hub.docker.com/r/siliconlabsinc/openthread-backbone-border-router/tags
+
+### Raspberry Pi set up
+
+1. On your SD card, make sure you flash the  [Raspbian Stretch Lite OS image](http://downloads.raspberrypi.org/raspbian_lite/images/raspbian_lite-2019-04-09/2019-04-08-raspbian-stretch-lite.zip) or [Raspbian Stretch with Desktop](https://downloads.raspberrypi.org/raspbian/images/raspbian-2019-04-09/2019-04-08-raspbian-stretch.zip).
+2. You can either SSH onto the Raspberry Pi or choose to work with the Raspbian Desktop directly. Open a terminal.
+3. Make sure to update your local repositories and package manager (**apt-get update** and **apt-get upgrade** prior to installing Docker).
+
+> Note: Make sure to reboot the Raspberry Pi after any updates.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:213]: Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 23]
```diff
@@ -0,0 +1,461 @@
...
+| 2     | `dataset masterkey 81ae2c2c17368d585dee71eaa8cf1e90` | Only the Master Key is required for a device to attach to a Thread network. | Done              |
+| 3     | `dataset commit active`                              | Commit new dataset to the Active Operational Dataset.        | Done              |
+| 4     | `ifconfig up`                                        | Enable Thread interface.                                     | Done              |
+| 5     | `thread start`                                       | Enable and attach Thread protocol operation.                 | Done              |
+|       |                                                      | Wait 20 seconds while the device joins and configures itself. |                   |
+| 6     | `state`                                              | View network configuration.                                  | child<br/>Done    |
+
+> Note: Because of the self configuring nature of a Thread network and these being Full Thread devices, either or both the FTDs may eventually become  routers. This can be found out using the `state` command.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:336]: Add this before the note

<noparse>> aside positive</noparse>
<noparse>></noparse>

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 24]
```diff
@@ -0,0 +1,461 @@
...
+
+2. BRD4166a
+
+3. BRD4168a
+
+4. BRD4180a
+
+5. BRD4304a
+
+   This list may be updated in the future SDK releases to include more radio boards. Please refer to the Silicon Labs OpenThread SDK release notes under Documentation for a full list of supported parts.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:422]: Don't indent this line, the parser thinks it's part of the list.

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 25]
```diff
@@ -0,0 +1,461 @@
...
+
+Flash the following demos on the respective boards. To flash, select your board under Debug Adapters on the left and click on RUN for the corresponding sample applications. A pop up window will show the flash progress.
+
+1. BRD4166A: **ot-rcp**
+
+   This device will function as a Radio Co Processor to the OT Border Router. We will use this device to create a Thread network and onboard the other two devices on the Thread network. Being a Border Router this device also acts as a gateway for the devices in the Thread network to communicate over the internet.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:428]: Can you have the text as part of the list item rather than separated?  So like

1. BRD4166A: **ot-rcp** — This device will...

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 26]
```diff
@@ -0,0 +1,461 @@
...
+
+2. Two BRD4168A: **ot-cli-ftd**
+
+   These two devices will act as Full Thread Devices. They will join the Thread network created by the OTBR.
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:432]: Same thing here as above

**Review by Vyrastas — CHANGES_REQUESTED** [review]
Thanks for all the updates.  A few more comments... most are from a fix to the note formatting, I gave you the wrong guidance before (sorry!).

I think we'll be good after this.

**Comment by silabs-mithil** [comment]
@Vyrastas Thank you once again for all the suggestions. I have revised the draft to incorporate them. Also moved the [status] to publish and added a button to download Studio on line 83

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md (hunk 8 above)
Comment (Vyrastas) [reply site/en/codelabs/silabs-openthread-hardware/index.lab.md:14]: Link still points to openthread rather than ot-docs

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 27]
```diff
@@ -0,0 +1,475 @@
+---
+id: silabs-openthread-hardware
+summary: In this Codelab, you'll program OpenThread on real hardware, create and manage a Thread network, and pass messages between nodes.
+status: [publish]
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:4]: The status should be [final]

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md (hunk 6 above)
Comment (Vyrastas) [reply site/en/codelabs/silabs-openthread-hardware/index.lab.md:24]: This was not done

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 28]
```diff
@@ -0,0 +1,475 @@
...
+### Communication between Thread devices
+
+We will use the `ping` command to check if devices can communicate with each other. To use the ping command we need the IPv6 addresses of the devices. These can be obtained using the `ipaddr` command.
+
+```console
+> ipaddr
+fd5c:c6b:3a17:40b9:0:ff:fe00:fc00					# Leader Anycast Locator (ALOC)
+fd5c:c6b:3a17:40b9:0:ff:fe00:1800					# Routing Locator (RLOC)
+fd5c:c6b:3a17:40b9:84e2:bae8:bd5b:fa03		# Mesh-Local EID (ML-EID)
+fe80:0:0:0:c449:ca4a:101f:5d16						# Link-Local Address (LLA)
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:365]: Can you align all the comments here to the Mesh-Local EID one so they don't wrap?

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md [hunk 29]
```diff
@@ -0,0 +1,475 @@
...
+```
+
+From both FTDs ping the OTBR using the OTBR's RLOC address.
+
+```console
+ping fd5c:c6b:3a17:40b9:0:ff:fe00:1800
+Done
+> 
+> 16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=30ms
+16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=52ms
```
Comment (Vyrastas) [inline site/en/codelabs/silabs-openthread-hardware/index.lab.md:375]: If this is using the OT CLI then ping should have the > prompt... can you double-check the output?  My understanding is that it would look like this, but I can't quickly test at the moment:

```
> ping fd5c:c6b:3a17:40b9:0:ff:fe00:1800
> 16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=30ms
16 bytes from fd5c:c6b:3a17:40b9:0:ff:fe00:1800: icmp_seq=3 hlim=64 time=52ms
```

**Review by Vyrastas — CHANGES_REQUESTED** [review]
Just a few more things!  After this we should be good to go.

File: site/en/codelabs/silabs-openthread-hardware/index.lab.md (hunk 29 above)
Comment (silabs-mithil) [reply site/en/codelabs/silabs-openthread-hardware/index.lab.md:375]: Yes, > is present for the CLI

**Comment by silabs-mithil** [comment]
Revised to correct the last few errors

**Review by Vyrastas — APPROVED** [review]
LGTM, thanks!

**Review by jwhui — APPROVED** [review]
Looks great. Thanks! 👍🏼

**Comment by Vyrastas** [comment]
@silabs-mithil I'll get this published on openthread.io tomorrow.

**Comment by Vyrastas** [comment]
@silabs-mithil This has been published:

https://openthread.io/guides#hardware_codelabs
https://openthread.io/codelabs/silabs-openthread-hardware

If anything needs fixing, please do it here in GitHub and we'll import the update to openthread.io.  Thanks for the contribution!

**Comment by silabs-mithil** [comment]
Looks good, thank you @Vyrastas and @jwhui
